test/integration/069_build_test/tests-failing/test.yml

```diff
@@ -0,0 +1,18 @@
+version: 2
+
+models:
+  - name: model_0
+    columns:
+      - name: iso3
+        tests:
+          - unique
+          - not_null
```

Comment:
do these create the tests with the graph? 

Maybe we should add a readme to this test to describe what's going on.